src/theory/uf/eq_proof.cpp

```diff
@@ -480,7 +480,7 @@ bool EqProof::expandTransitivityForDisequalities(
     p->addStep(congConclusion,
                PfRule::CONG,
                substPremises,
-               {nm->operatorOf(kind::EQUAL)},
+               {ProofRuleChecker::mkKindNode(kind::EQUAL)},
```

Comment:
This breaks the version of CONG on master, but since this is not active code yet, maybe its ok.  I will submit the update to CONG in the meantime.